macos/docs/Controls/Button.md

```diff
@@ -0,0 +1,37 @@
+# Button
+
+## Overview
+Buttons are one of the core controls that make an app feel native to the platform it's on. It’s important to respect the platform's paradigms in order to help the user feel at home on macOS and keep the experience quality high.
```

Comment:
I don't have a great idea of how to improve this, but I feel like the overview should at least in some way touch on what the control actually is/does. Maybe we can take some inspiration from the Web button overview? Note some things are different here, especially their focus on navigation, because of web paradigms. 

https://developer.microsoft.com/en-us/fluentui#/controls/web/button

Or maybe the NSButton docs are a better starting point: https://developer.apple.com/documentation/appkit/nsbutton

Maybe something like: 

> Buttons are a standard control that users can interact with to take actions within your application.

would be sufficient. 

https://developer.microsoft.com/en-us/fluentui#/controls/web/button

Reply:
This snippet of text was what I got from the iOS and Android. I'm not particularly happy or beholden to it. I'll add some extra info on how we subclass NSButton but also add hover effects in comformance to macOS (since I literrally use the system effect for [hover](https://developer.apple.com/documentation/appkit/nscolor/systemeffect/rollover) to color that state)